ocp_utilities/infra.py

```diff
@@ -49,6 +55,32 @@ def assert_nodes_schedulable(nodes):
         )
 
 
+def assert_pods_failed_or_pending(pods: list[Pod]) -> None:
+    """
+    Validates all pods are not in failed nor pending phase
+
+    Args:
+         pods: List of pod objects
+
+    Raises:
+        PodsFailedOrPendingError, if there are failed or pending pods
```

Comment:
`PodsFailedOrPendingError,` should be `PodsFailedOrPendingError: if there are failed or pending pods`

Reply:
Done